webapp/app.py

```diff
@@ -22,12 +23,11 @@
     }
 )
 
-docs = FlaskApiSpec(app)
-
-
-@app.route("/hello-world")
-def get_hello_world():
-    return jsonify({"message": "Hello World!"})
-
+app.add_url_rule(
+    "/cves/<cve_id>",
```

Comment:
Let''s please make this end with `.json`, but happy for that to happen in a follow-up PR.